css2/Overview.bs

```diff
@@ -16753,12 +15676,9 @@ Colors</a></h3>
 
 <h3 id="changes">Changes</h3>
 
-<!--
-<h3 id="c1.1"><a href="about.html#css2.1-v-css2">Section 1.1 CSS 2.1 vs CSS 2</a></h3>
```

Comment:
GitHub blame fails at letting me know when this id was added on line 16757 - any concerns about breaking links to it? Also, I'm wondering if any/all of the ids in the removed section should be added to this change entry, so that links to the removed section go here?

Reply:
We dropped the changes section pretty early on, and decided not to care about links to it. We can re-add at the least the 2016 anchors for all the 2.1 changes, but the previous decision had been we don't need to bother keeping them.

Reply:
(Feel free to open a new issue if you believe we should maintain all the IDs that appear in the Changes section in 2.1 or if we should maintain the entire Changes list from 2.0)